configs/pointnet2/pointnet2_msg_16x2_scannet-3d-20class.py

```diff
@@ -0,0 +1,36 @@
+_base_ = [
+    '../_base_/datasets/scannet_seg-3d-20class.py',
+    '../_base_/models/pointnet2_msg.py', '../_base_/default_runtime.py'
+]
+
+# data settings
+data = dict(samples_per_gpu=16)
+evaluation = dict(interval=5)
+
+# model settings
+model = dict(
+    decode_head=dict(
+        num_classes=20,
+        ignore_index=20,
+        loss_decode=dict(class_weight=[
+            2.389689, 2.7215734, 4.5944676, 4.8543367, 4.096086, 4.907941,
+            4.690836, 4.512031, 4.623311, 4.9242644, 5.358117, 5.360071,
+            5.019636, 4.967126, 5.3502126, 5.4023647, 5.4027233, 5.4169416,
+            5.3954206, 4.6971426
+        ])),
```

Comment:
How to determine these class_weight? Is there any convention? Is there any need for comments?

Reply:
Yes I think I need to add some comments here. Basically, this weight is generated in data_processing, saved in `class_weight.npy`. However, in order to pass the Github CI, I replace the weight file path with hard-coded values.

Reply:
Yes I think I need to add some comments here. Basically, this weight is generated in data_processing, saved in `class_weight.npy`. However, in order to pass the Github CI, I replace the weight file path with hard-coded values.